src/brasil/gov/tiles/tiles/templates/highlightscarousel.pt

```diff
@@ -26,7 +26,7 @@
               <tal:image define="scales obj/@@images;
                                  thumbnail python:scales.scale('image', scale=None)"
                          condition="thumbnail">
-                <img tal:attributes="src thumbnail/url;
+                <img tal:attributes="src thumbnail/url|string:${obj/absolute_url}/download;
```

Comment:
@claytonc this fixes the issue with Archetypes-based images; I took this from the [carousel tile template](https://github.com/collective/collective.cover/blob/2.0b1/src/collective/cover/tiles/templates/carousel.pt#L31).